pkg/controllers/workapplier/status.go

```diff
@@ -832,3 +854,15 @@ func setWorkStatusTrimmedCondition(work *fleetv1beta1.Work, sizeDeltaBytes, size
 		ObservedGeneration: work.Generation,
 	})
 }
+
+func shouldSkipStatusUpdate(isDriftedOrDiffed, isStatusBackReportingOn bool, originalStatus, currentStatus *fleetv1beta1.WorkStatus) bool {
+	if isDriftedOrDiffed || isStatusBackReportingOn {
```

Comment:
this will reduce the effectiveness of this PR by a lot, is there anyway to soften the blow a bit further?

Reply:
Hi Ryan! We might need a flag to omit observation timestamps if needed? Though with exponential backoff, as long as the number of drifted/diffed placements + status back-reporting placements is low, and the system does not restart often, the # of calls should be relatively limited (~96 writes per placement) when everything stablizes.